### PR TITLE
Update requester invite approved email

### PR DESCRIPTION
**Pull request by spatel033**
When a user successfully requests to join a service they should get an email request has been approved. More details on [this card](https://trello.com/c/FHhogwve/904-send-a-different-invite-email-if-the-user-has-requested-to-join-a-service).

### CODE REVIEW

File: migrations/versions/0464_requester_invite_approved.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            false,
+            current_timestamp,
+            '{current_app.config["NOTIFY_USER_ID"]}'
+        )
+        ON CONFLICT DO NOTHING
+        """
+    )
+
+
+def downgrade():
```
Comment (rparke) [inline migrations/versions/0464_requester_invite_approved.py:88]: Shouldn't DB migrations always have a downgrade?

File: migrations/versions/0464_requester_invite_approved.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Sign in to GOV.UK Notify to get started.
+
+Thanks
+
+GOV.​UK Notify
+https://www.gov.uk/notify"""
+)
+
+
+def upgrade():
```
Comment (rparke) [inline migrations/versions/0464_requester_invite_approved.py:31]: If we are just adding DB records, and not changing the DB schema, (unless there's some subtlety I'm not getting which ... there may be), adding records should just be done _via_ DAO's.

File: app/service_invite/rest.py [hunk 3]
```diff
@@ -31,36 +31,25 @@
...
-            ),
-        },
-        notification_type=EMAIL_TYPE,
-        api_key_id=None,
-        key_type=KEY_TYPE_NORMAL,
-        reply_to_text=invited_user.from_user.email_address,
-    )
-
-    send_notification_to_queue(saved_notification, queue=QueueNames.NOTIFY)
+    if is_join_request:
```
Comment (leohemsted) [inline app/service_invite/rest.py:44]: given this flow's so different

"create an invite in the database, send that user an email with a link to join service"

"vs person has already been added to service, give them a confirmation email saying so"

i feel like these could be separate endpoints?

that would make this a lot tidier and much easier to reason about when looking at eg access logs

**Review by leohemsted — COMMENTED** [review]
does this work? it used to send an invite, now it just tells them "you've been added to the service" but does it actually add them? does the request to join a service flow still work after this code goes in?

i need to have a think about this story overall - sorry but i think theres been some confusion, probably my fault, about whether this story should have been picked up at all.

**Comment by spatel033** [comment]
Closing because of requirement changes

**Pull request closed without merging**